lib/olive_branch_patch.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This will monkey patch olivebranch middleware https://github.com/vigetlabs/olive_branch/blob/master/lib/olive_branch/middleware.rb
+# so that when the VA inflection changes a key like something_va_something, the results will
+# have the old-inflection somethingVaSomething.
+
+module OliveBranchMiddlewareExtension
+  def call(env)
+    result = super(env)
+    _status, _headers, response = result
+    if env['HTTP_X_KEY_INFLECTION'] =~ /camel/i
+      response.each do |json|
+        # do not process strings that aren't json (like pdf responses)
+        next unless json.is_a?(String) && json.starts_with?('{')
```

Comment:
This doesn't prove that it's json and is janky.  you could just look at headers["Content-Type"]

Reply:
pushed an update.  uses `send` which is also a bit janky, but it relies on the same method that olive_branch uses to determine if it should transform any values.  What do you think?

Reply:
Thanks!  I prefer that!